web/crux-ui/locales/en/notifications.json

```diff
@@ -24,5 +24,13 @@
     "message": "Its a test!",
     "success": "Your hook is working!",
     "error": "Something is wrong with your hook!"
+  },
+
+  "events": "Send message when:",
+  "eventType": {
+    "deploymentCreated": "New deployment is created",
+    "versionCreated": "New version is added",
+    "nodeAdded": "New node is added",
+    "userTeamInvited": "New user invited to my team"
```

Comment:
"userInvited": "New user invited to the team"
@polaroi8d , @gerimate  ?

Reply:
+1 but, what if the user belongs to several teams ?

Reply:
Right. We should include the team name into the notification.

Reply:
Already included

Reply:
The exact message is : "XY has been invited to join team TEAM by XY !"